src/main/java/org/apache/cassandra/distributed/api/NodeToolResult.java

```diff
@@ -65,6 +74,16 @@ public Throwable getError()
         return error;
     }
 
+    public String getStdOut()
+    {
+        return capturedConsoleOut;
+    }
+
+    public String getStdErr()
```

Comment:
came comment as above, asserts are `stderr` but this is `stdErr`, should pick one and stick with it.